version: robot-Mec&4WD
port_name: /dev/ttyUSB2 #设备端口
baud_rate: 115200       #通信波特率，固定115200

#odom_frame: odom               #里程计坐标系
base_frame: base_footprint      #底盘坐标系
imu_frame: base_imu_link        #IMU坐标系

linear_correction_factor: 1.345  #线速度校准系数
angular_correction_factor: 0.8  #角速度校准系数

#publish_odom_transform: True   #是否发布odom->base_footprint坐标变换
Kp: 53                         #OpenCRP电机控制PID，P参数
Ki: 0                           #OpenCRP电机控制PID, I参数
Kd: 3                         #OpenCRP电机控制PID，D参数

control_rate: 50                #底盘控制频率  
